api/get_rbg_hls_link.php

```diff
@@ -1,20 +1,26 @@
 <?php
 include 'simple_html_dom.php';
-include_once 'CacheAccess.php';
 
 function get_rbg_hls_link(string $url): ?string {
-    $html = CacheAccess::getHtml($url, 60 * 60 * 24); // this site should be static, therefore we cache it longer.
+    $CACHEKEY = "get_rbg_hls_link_" . $url;
+    $CACHETTL = 60 * 60 * 24; // One day
+    if (apcu_exists($CACHEKEY)) {
+        return apcu_fetch($CACHEKEY);
+    }
+
+    $html = file_get_html($url);
 
     if ($html) {
         $matches = [];
 
         preg_match('/MMstartStream.*\'(.+)\'/', $html, $matches);
 
         if (sizeof($matches) >= 1) {
+            apcu_add($CACHEKEY, $matches[0], $CACHETTL);
```

Comment:
Is there no chance that this has more than one match?

Reply:
If so, please explain why in a comment, so future generations of engineers don't have to guess :)

Reply:
I didn't write this myself so I was kind of irritated by this as well but then I figured that this file is only used in video_only.php and offers basically the same functionality as get_rbg_stream_site.php so I went ahead and deleted it entirely.

Reply:
"Ok" :man_shrugging: